CMSgov/easi-app
# [EASI-3187] - Update LCID

# EASI-3187

[Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=429-114324&mode=design&t=9mQLI3qT60QApN7q-0)

## Changes and Description

- Update LCID action form
- `LcidSummary` component - src/views/GovernanceReviewTeam/Actions/ManageLcid/LcidSummary.tsx
- `LcidStatusTag` component (used in `LcidSummary`) - src/components/shared/LcidStatusTag/index.tsx
  - Confirmed additional statuses not shown on Figma with Zoe:
    - Expiring soon - 60 days from expire date
    - Retiring soon - 60 days from retire date
- Field validation - user must fill at least one LCID field (above divider in form) 



## How to test this change

1. Sign in with `itGovV2Enabled` flag on
2. Issue LCID
    - Go to /governance-review-team/:systemId/actions
    - Select "Issue a decision or close request"
    - Select "Issue a Life Cycle ID" and complete form
3. Test Update LCID form
    - Select "Manage a Life Cycle ID"
    - Select "Update LCID"
    - Ensure form works as expected
      - Check that error message shows if LCID fields left blank
 
## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
